frontend/src/reports/TwoVariableReport.tsx

```diff
@@ -168,47 +171,93 @@ function TwoVariableReport(props: {
       {DEMOGRAPHIC_BREAKDOWNS.map((breakdownVar) =>
         !breakdownIsShown(breakdownVar) ? null : (
           <>
-            {variableConfig1.metrics["pct_share"] && (
-              <Grid item xs={12} sm={6}>
+            <RowOfOptionalMetrics
+              metric1={variableConfig1.metrics["pct_share"]}
+              metric2={variableConfig2.metrics["pct_share"]}
+              fips1={props.fips1}
+              fips2={props.fips2}
+              createCard={(
+                metricConfig: MetricConfig,
+                fips: Fips,
+                unusedUpdateFips: (fips: Fips) => void
+              ) => (
                 <DisparityBarChartCard
-                  metricConfig={variableConfig1.metrics["pct_share"]}
+                  metricConfig={metricConfig}
                   breakdownVar={breakdownVar}
-                  fips={props.fips1}
+                  fips={fips}
                 />
-              </Grid>
-            )}
-            {variableConfig2.metrics["pct_share"] && (
-              <Grid item xs={12} sm={6}>
-                <DisparityBarChartCard
-                  metricConfig={variableConfig2.metrics["pct_share"]}
-                  breakdownVar={breakdownVar}
-                  fips={props.fips2}
-                />
-              </Grid>
-            )}
-            {variableConfig1.metrics["per100k"] && (
-              <Grid item xs={12} sm={6}>
-                <SimpleBarChartCard
-                  metricConfig={variableConfig1.metrics["per100k"]}
-                  breakdownVar={breakdownVar}
-                  fips={props.fips1}
-                />
-              </Grid>
-            )}
-            {variableConfig1.metrics["per100k"] && (
-              <Grid item xs={12} sm={6}>
+              )}
+            />
+            <RowOfOptionalMetrics
+              metric1={variableConfig1.metrics["per100k"]}
+              metric2={variableConfig2.metrics["per100k"]}
+              fips1={props.fips1}
+              fips2={props.fips2}
+              createCard={(
+                metricConfig: MetricConfig,
+                fips: Fips,
+                unusedUpdateFips: (fips: Fips) => void
+              ) => (
                 <SimpleBarChartCard
-                  metricConfig={variableConfig2.metrics["per100k"]}
+                  metricConfig={metricConfig}
                   breakdownVar={breakdownVar}
-                  fips={props.fips2}
+                  fips={fips}
                 />
-              </Grid>
-            )}
+              )}
+            />
           </>
         )
       )}
     </Grid>
   );
 }
 
+function RowOfOptionalMetrics(props: {
```

Comment:
Would a better name for this be RowOfTwoOptionalMetrics to make it clear it's only for two variables?

Reply:
ya good idea